### PR TITLE
Hello from the future

### PR DESCRIPTION
**Pull request by MartinNievas**
This is a very good example of PR  👍

### CODE REVIEW

File: martin-ejercicio1.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+
+int main(int argc, char *argv[]) {
+	
+	
+	for(int i=-9; i < 10; i++){
+		if(i < 0)
+			printf("%d", -i);
+		else
+			printf("%d", i);
```
Comment (martinnievas20) [inline martin-ejercicio1.c:10]: you can add braces for this

**Review by martinnievas20 — COMMENTED** [review]
¬¬

File: martin-ejercicio1.c (hunk 1 above)
Comment (MartinNievas) [reply martin-ejercicio1.c:10]: ¬¬

Comment (martinnievas20) [reply martin-ejercicio1.c:10]: :V

**Pull request merged**